docs/chapters/infrastructure/ci-cd/host.rst

```diff
@@ -0,0 +1,91 @@
+Infrastructure host
+===================
+
+Because we run all the parts of the infrastructure like code hosting, cache, documentation, on different software and probably in docker containers there needs to be one host server which will work as a proxy server to give easy access to all the applications in the infrastructure in a easy way. It might also be responsible to serve static files like documentation and cache.
```

Comment:
Same reasoning about style here. Should we be more concrete in what will be used or do we leave that to some description of a concrete setup, i.e. in a SWF Deployment doc?